utils.go

```diff
@@ -66,3 +83,99 @@ func IsBinary(data []byte) bool {
 
 	return true
 }
+
+// LoadGitattributes reads and parse the file .gitattributes wich overrides the standards strategies
+func LoadGitattributes() {
+	rawAttributes, err := loadGitattributes(".gitattributes")
+	if err == nil && len(rawAttributes) > 0 {
+		parseAttributes(rawAttributes)
+	}
+}
+
+func loadGitattributes(name string) (map[string]string, error) {
```

Comment:
Ignore me if this is some kind of Go convention, but having 2 functions whose only difference in the name is the case of the first letter seems pretty confunsing to me